packages/jest-circus/src/eventHandler.ts

```diff
@@ -12,7 +12,7 @@ import {
 } from './globalErrorHandlers';
 import {TEST_TIMEOUT_SYMBOL} from './types';
 import {
-  addErrorToEachTestUnderDescribe,
+  addErrorToEachChildUnderDescribe,
```

Comment:
Renamed to `addErrorToEachChildUnderDescribe` because errors are being sent to child describe blocks as well.